### PR TITLE
Niliev/avds

### PR DESCRIPTION
**Pull request by NickIliev**
Rewritten article about creating Andriod Virtual Devices using the latest GUI in Android Studio.

### CODE REVIEW

File: tooling/android-virtual-devices.md [hunk 1]
```diff
@@ -7,68 +7,49 @@ slug: android-emulators
...
 ## Create a new emulator 
 
-1. Open **Android SDK Manager** 
+Open **Android Studio** and start AVD manager located in `Tools > Android > AVD Manager`
 
-     - From your terminal run  `start %ANDROID_HOME%/"SDK Manager.exe"`
+![Navigate to AVD Manager](../img/create-avd/avd_001.png "Navigate to AVD Manager")
 
-2. Make sure that the system images for the AVDs are installed. 
+The AVD Manager will load the list of already installed system images (if any) and will also provide you with an option to create new emulator via `Create Virtual Device button located at the bottom left corner.
```
Comment (tjvantoll) [inline tooling/android-virtual-devices.md]: You have an unclosed quote here with “Create Virtual Device”.

File: tooling/android-virtual-devices.md [hunk 2]
```diff
@@ -92,7 +73,7 @@ e.g. run on device with Device Identifier
 tns run android --device 00bd261c1580a7d3
 ```
 
-The example above will start the application on device with id 2 (sdk_phone_x86_64).
+The example above will start the application on the device with id 2 (sdk_phone_x86_64).
 
 To launch a NativeScript application on all connected devices simultaneously, simply ignore the `--device <id>` flag.
```
Comment (tjvantoll) [inline tooling/android-virtual-devices.md:86]: Not related to this change, but I don’t believe you can launch a {N} app on all connected devices simultaneously anymore.

Comment (tjvantoll) [reply tooling/android-virtual-devices.md:86]: Might be worth noting the default behavior if you use `tns run android` without the `--device` flag. (I believe the CLI picks the first device on the list.)

File: tooling/android-virtual-devices.md [hunk 3]
```diff
@@ -7,68 +7,49 @@ slug: android-emulators
...
 
-    ![Launched AVD](../img/create-avd/avd-004.png "Launched AVD")
 
 # Using the emulators with NativeScript
 
-Once the emulator is launched, we can start the development process with NativeScript.
-In many occasions the developers would like to work with several emulators simultaneously.
-The best practice for this scenarios is to have all emulators opened in advance (to avoid launch timings and possible timeouts during build).
-Once all the selected emulators are launched and all the real devices are attached we can check their status.
+Once the emulator is launched, we can start the development process with NativeScript. In many occasions, the developers would like to work with several emulators simultaneously. The best practice for this scenarios is to have all emulators opened in advance (to avoid launch timings and possible timeouts during build). Once all the selected emulators are launched, and all the real devices are attached we can check their status.
```
Comment (tjvantoll) [inline tooling/android-virtual-devices.md]: > Once the emulator is launched, we can start the development process with NativeScript. In many occasions, the developers would like to work with several emulators simultaneously.

Not related to this PR, but I’d like to suggest a rewording of this.

Once you launch an emulator, you’re ready to start developing NativeScript apps on this emulated device. Oftentimes during development you’ll want to work with several emulators simultaneously. The best practice for this scenario is to have all emulators...

**Review by tjvantoll — COMMENTED** [review]
Looks good 👍 

I just had a few relatively trivial notes.

File: tooling/android-virtual-devices.md [hunk 4]
```diff
@@ -7,68 +7,49 @@ slug: android-emulators
...
-    - **Keyboard** : Check "hardware keyboard present"
-    - **Skin** : Choose the skin type for the emulator
-    - **Front & Back Camera** : Enable/disable support for emulated or web cameras
-    - **Memory Options** : Recommended to leave this as set by default!
-    - **Internal Storage** : Default value 200MB - Increase this value to ensure that there is enough space for the applications that will be deployed later.
-    - Check "Use Host GPU" to improve performance if your GPU provides enough resources.
+![System Image Next Screen](../img/create-avd/avd_006.png "System Image")
 
-    ![AVD creation tool](../img/create-avd/avd-003.png "AVD creation tool")
+The final configuration screen will load. If fine tuning is needed, here we can change the default settings of the new emulator (e.g. default screen orientation, device frame, memory options, etc.). When all settings are set and ready press `Finish`.
```
Comment (PetyaSotirova) [inline tooling/android-virtual-devices.md]: If fine-tuning is needed, you can change...

**Review by PetyaSotirova — APPROVED** [review]
The article has a lot of images, consider adding numbers and captions, e.g. https://docs.nativescript.org/tooling/chrome-devtools.
You can then refer to the figure number:
"In the screen above, an Android 6 Google APIs system image...' -> "In Figure n, an Android 6 Google APIs..."

File: tooling/android-virtual-devices.md (hunk 2 above)
Comment (NickIliev) [reply tooling/android-virtual-devices.md:86]: @tjvantoll running on multiple devices is still the default behavior - if you have multiple Android emulators/devices `tns run` will deploy the application on each of them.

Comment (tjvantoll) [reply tooling/android-virtual-devices.md:86]: Ah, good to know 😄

**Review by vchimev — APPROVED** [review]

**Pull request merged**